File: android/DFM/src/main/java/com/dontflymoney/api/Site.java
package com.dontflymoney.api;

public class Site
{
	private static final String publicDomain = "dontflymoney.com";
	static final String Domain = publicDomain;
	
	static String GetProtocol()
	{
		return IsLocal() ? "http" : "https";
	}
	
	public static Boolean IsLocal()
	{
		return !Domain.equals(publicDomain);
	}
	
	
}

File: android/DFM/src/main/kotlin/com/darakeon/dfm/api/Site.kt
package com.darakeon.dfm.api

object Site {
	private val publicDomain = "dontflymoney.com"
	internal val Domain = publicDomain

	internal fun GetProtocol(): String = if (isLocal) "http" else "https"

	private val isLocal: Boolean get() = Domain != publicDomain


}
